internal/kms/kms.go

```diff
@@ -331,11 +332,13 @@ func (c *kmsConn) ListKeyNames(ctx context.Context, req *ListRequest) ([]string,
 		return nil, "", errListingKeysFailed(err)
 	}
 
-	names := make([]string, 0, len(resp.Items))
-	for _, item := range resp.Items {
-		names = append(names, item.Name)
+	keyInfos := make([]madmin.KMSKeyInfo, len(resp.Items))
+	for i, v := range resp.Items {
+		keyInfos[i].Name = v.Name
+		keyInfos[i].CreatedAt = fmt.Sprintf("%d", v.CreatedAt.Unix())
```

Comment:
I'm using unix timestamp for this format, let me know if it should be something else.

Reply:
Yes we can, it just pushes the conversion to a `string` out into the console, which ultimately needs to do so in order to render the JSON response. The PR to make this change is here: https://github.com/minio/console/pull/3421/files

KMSKeyInfo.CreatedAt is a string, but so are all the other `CreatedAt` fields for those KMS structs. To keep them consistent, this PR changes all of them to `time.Time`: https://github.com/minio/madmin-go/pull/296/files

Let me know your thoughts, I think it's good to be consistent in a SDK. It doesn't look like those other ones are used, and might be dead code? For example I can't find a `/minio/kms/v1/policy` or `/minio/kms/v1/identity` route in the MinIO server, yet madmin has methods for those API calls. Am I missing something here?